src/tinyspline.h

```diff
@@ -722,11 +738,11 @@ size_t ts_deboornet_multiplicity(const tsDeBoorNet *net);
  * 	The number of insertions that were necessary to evaluate the knot of \p
  * 	net.
  */
-size_t ts_deboornet_num_insertions(const tsDeBoorNet *net);
+size_t TINYSPLINE_API ts_deboornet_num_insertions(const tsDeBoorNet *net);
 
 /**
  * Returns the dimension of \p net. The dimension of a net describes the number
- * of components for each point in ts_bspline_get_points(spline).
+ * of components for each point in TINYSPLINE_API ts_bspline_get_points(spline).
```

Comment:
search and replace fail? :)

Reply:
You caught me red handed. 😄 
I did `s/ (ts_[a-z]+)/TINYSPLINE_API \1/`